retrieval_service/datastore/providers/postgres.py

```diff
@@ -215,14 +218,76 @@ async def initialize_data(
                   departure_airport TEXT,
                   arrival_airport TEXT,
                   departure_time TIMESTAMP,
-                  arrival_time TIMESTAMP
+                  arrival_time TIMESTAMP,
+                  seat_row INTEGER,
+                  seat_letter TEXT
+                )
+                """
+            )
+            # Insert all the data
+            await conn.executemany(
+                """INSERT INTO tickets VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12)""",
+                [
+                    (
+                        t.id,
+                        t.user_id,
+                        t.user_name,
+                        t.user_email,
+                        t.airline,
+                        t.flight_number,
+                        t.departure_airport,
+                        t.arrival_airport,
+                        t.departure_time,
+                        t.arrival_time,
+                        t.seat_row,
+                        t.seat_letter,
+                    )
+                    for t in tickets
+                ],
+            )
+
+            # If the table already exists, drop it to avoid conflicts
+            await conn.execute("DROP TABLE IF EXISTS seats CASCADE")
+            # Create a new table
+            await conn.execute(
+                """
+                CREATE TABLE seats(
+                  flight_id INTEGER,
```

Comment:
Nit: Does this flight_id corresponds to the `id` column in flights table? I'm wondering will it be better to update the `id` column to `flight_id` for flights database. (not sure how much this will help the llm agent to understand things better).

Reply:
I think it's normal and best practice to have "id" in a table x end up being "x_id" in other tables. Same with ticket_id below.

Reply:
Got it~ sounds good!